config/notify.config.js

```diff
@@ -5,13 +5,16 @@
  * @module NotifyConfig
  */
 
+const defaultBatchSize = 250
+
 // We require dotenv directly in each config file to support unit tests that depend on this subset of config.
 // Requiring dotenv in multiple places has no effect on the app when running for real.
 require('dotenv').config()
 
 const config = {
   apiKey: process.env.GOV_UK_NOTIFY_API_KEY,
-  stubNotify: process.env.STUB_NOTIFY || true // Used to perform integration tests with notify
+  batchSize: process.env.NOTIFICATIONS_BATCH_SIZE || defaultBatchSize,
```

Comment:
I'm fine to put this inline. Where we have defaults in other places in the config, we do this.

```suggestion
  batchSize: process.env.NOTIFICATIONS_BATCH_SIZE || 250,
```

Else, if you want to keep the named param we should change the param to reflect it is a constant; `DEFAULT_BATCH_SIZE`.

I also think it would be helpful to have add a `// NOTE: ` above the line to copy the bit from your explanation on where 250 comes from.

Reply:
Sonar complained about the 'magic number' hence it being moved up.

Reply:
inline added and comment added based on commit message with link provided to docs.